osp/views/answer.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import viewsets, status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from osp.models import (
+    Question, Answer, ChoiceValue, CheckboxValue, DropdownValue, DateValue,
+    TimeValue, FileUploadValue, ShortAnswerValue, ParagraphValue, FormFeedback
+)
+from osp.serializers.answer import AnswerSerializer
+from osp.serializers.form_feedback import FormFeedbackSerializer
+from osp.utils.answer_function import get_model_and_serializer
+
+class AnswerView(viewsets.ModelViewSet):
+
+    serializer_class = AnswerSerializer
+    permission_classes = [IsAuthenticated]
+    queryset = Answer.objects.all()
+
+    def get_queryset(self):
+        queryset = self.queryset
+        form_id = self.request.query_params.get('form_id', None)
+        user = self.request.user
+        if form_id and user:
+            queryset = queryset.filter(form_answers__user_id=user.id, form_answers__form_id=form_id)
+        return queryset
```

Comment:
return the ordered queryset by the `order` field on questions. Preferably on the line 17 i.e. defining queryset